app/clients/letter/dvla.py

```diff
@@ -0,0 +1,71 @@
+import time
+
+import boto3
+import jwt
+import requests
+from flask import current_app
+
+
+class SSMProperty:
+    def __init__(self, key):
+        self.key = key
+
+    def __get__(self, obj, objtype=None):
+        return obj.ssm_client.get_parameter(Name=self.key, WithDecryption=True)["Parameter"]["Value"]
+
+    def __set__(self, obj, value):
+        # this errors if the parameter doesn't exist yet in SSM as we haven't supplied a `Type`
+        # this is fine for our purposes, as we'll always want to pre-seed this data.
+        obj.ssm_client.put_parameter(Name=self.key, Value=value, Overwrite=True)
+
+
+class DVLAClient:
+    """
+    Amazon SES email client.
+    """
+
+    ssm_client = None
+    statsd_client = None
+
+    _jwt_token = None
+    _jwt_expires_at = None
+
+    dvla_username = SSMProperty(key="/notify/api/dvla_username")
+    dvla_password = SSMProperty(key="/notify/api/dvla_password")
+    dvla_api_key = SSMProperty(key="/notify/api/dvla_api_key")
+
+    def init_app(self, region, statsd_client):
+        self.ssm_client = boto3.client("ssm", region_name=region)
+        self.statsd_client = statsd_client
+        self.request = requests.Session()
+
+    @property
+    def name(self):
+        return "dvla"
```

Comment:
What's this for? 🤔

Reply:
this was for consistency with the MMGClient, FiretextClient, and AWSSesClient: we have a dict where we look-up by name here:

https://github.com/alphagov/notifications-api/blob/075d0c92f4366da666893730f32ee6923434b7f4/app/clients/__init__.py#L22

I'm not 100% sold on this being necessary for letters though